### PR TITLE
Update gauntlet v0.2 to reflect results of calibration

### PR DESCRIPTION
**Pull request by bmosaicml**
In order to filter out noisy tasks and set appropriate number of few shot per task, we used the following methodology:

1. Evaluate each benchmark on several 3B models trained for [20x, 50x, 100x, 250x, 500x] chinchilla token duration for  num_fewshot=[0, 1, 3, 5, 10]
2. Determine which num_fewshot settings the models have monotonically increasing performance. If multiple num_fewshot have monotonically increasing performance we broke ties by choosing the industry standard if there is one, or `3`.
3. For benchmarks which were scoring 0 (and therefore not monotonic), we excluded them unless they are known to show good scaling at larger model scales. These tasks will average in as 0 in expectation, but they will introduce slight variance.

Later versions of this experiment will estimate the exact confidence intervals of the gauntlet sub-scores + overall average.

image

[Google sheet](https://docs.google.com/spreadsheets/d/1rkM6gBc2yX1qYbRjuVlhR9CFr2gOsKNENZgW71X5LnM/edit#gid=0)

### CODE REVIEW

File: scripts/eval/yamls/eval_gauntlet_v0.2.yaml [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+      random_baseline: 0.0
+    - name: simple_arithmetic_nospaces
+      num_fewshot: 5
+      random_baseline: 0.0
+    # - name: math_qa
+    #   num_fewshot: 10
+    #   random_baseline: 0.25
+    # - name: logi_qa
+    #   num_fewshot: 10
+    #   random_baseline: 0.25
```
Comment (abhi-mosaic) [inline scripts/eval/yamls/eval_gauntlet_v0.2.yaml:92]: should we comment or should we delete?

**Review by abhi-mosaic — APPROVED** [review]
small comment, but otherwise looks good to me! I spot checked the most important ICLs and the k-shots look like what was discussed.

File: scripts/eval/yamls/eval_gauntlet_v0.2.yaml (hunk 1 above)
Comment (bmosaicml) [reply scripts/eval/yamls/eval_gauntlet_v0.2.yaml:92]: I'll delete

**Review by dakinggg — APPROVED** [review]
LGTM from my perspective. Please wait for whoever you want to take a look from research.

**Review by tbarton16 — APPROVED** [review]
LGTM

**Review by sashaDoubov — APPROVED** [review]
LGTM 🚀

**Review by mansheej — APPROVED** [review]

**Pull request merged**